README.md

```
@@ -0,0 +1,94 @@
+# 프로젝트 명: One Ticket
+
+## 프로젝트 소개
+
+**실시간 예매 처리를 위한 MSA 기반 티켓팅 시스템**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)